tcpclient/connection.go

```diff
@@ -71,3 +85,26 @@ func PrintGroupOfConnections(c []Connection) string {
 	return fmt.Sprintf("Total: %d, Dialing: %d, Established: %d, Closed: %d, Error: %d, NotInitiated: %d",
 		nTotal, nDialing, nEstablished, nClosed, nError, nNotInitiated)
 }
+
+func PrintFinalMetricsReport(c []Connection) string {
```

Comment:
i don't think this is the right place for this function. Have you think about moving it to mtcpclient/reporting.go ? mtcpclient is the one who knows about groups of connections, and it already has some reporting capabilities, but the tcpclient/connection functionality, that works at lower level...

Reply:
👍